src/llmcompressor/modifiers/awq/base.py

```diff
@@ -131,9 +132,11 @@ class AWQModifier(Modifier, QuantizationMixin):
 
     # Private vars set during initialization, cleared during finalization
     _resolved_mappings: List[ResolvedMapping] = PrivateAttr(default_factory=list)
-    _activations: Dict[str, List[torch.Tensor]] = PrivateAttr(default_factory=dict)
-    _activation_hooks: Set[RemovableHandle] = PrivateAttr(default_factory=set)
-    _module_kwargs: Dict = PrivateAttr(default_factory=dict)
+    _samples: Dict[Module, IntermediatesCache] = PrivateAttr(
+        default_factory=IntermediatesCache
+    )
+    _sample_means: Dict[Module, float] = PrivateAttr(default_factory=dict)
+    _num_samples: Dict[Module, int] = PrivateAttr(default_factory=dict)
```

Comment:
I know this is in GPTQ, but a field named `num_samples` indicates an `int` in any virtually context i've seen it. what bout `_sample_counts` so it's more similar to `sample_means`?

Reply:
That's good too!